modules/sdk/project-templates-npm-angular-portlet/src/main/resources/archetype-resources/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -2,13 +2,8 @@
 
 <%@ include file="/init.jsp" %>
 
-<p>
-	<b><liferay-ui:message key="${artifactId}.caption"/></b>
-</p>
-
-<div id="${artifactId}-root">
-</div>
+<div id="<portlet:namespace />"></div>
```

Comment:
Isn't it strange to have a `<div>` with an ID equal to the portlet namespace, without anything after it? Idk, something like `<portlet:namespace />wrapper`...

Reply:
Well the portlet:namespace is references below in the ${auiScriptRequireVarName}.default('<portlet:namespace />');
right?

Reply:
Yes, it just looks a bit strange... But okay, I think it's no big deal!